src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx

```diff
@@ -26,7 +27,12 @@ export const MetaNavigation = connect(function MetaNavigation({
     )
   }
 
-  const sitePortalOverviewPage = root.get('sitePortalOverviewPage')
+  const rawSitePortalOverviewPage = root.get('sitePortalOverviewPage')
+  // TODO: Remove workaround, once #10699 is available
+  const sitePortalOverviewPage =
+    (isRedirect(rawSitePortalOverviewPage) &&
+      rawSitePortalOverviewPage.get('link')?.obj()) ||
+    rawSitePortalOverviewPage
```

Comment:
* [ ] FYI Instead of repeating the same code three times, how about a dedicated (`getSitePortalOverviewPage`) or generic (`followRedirect(root.get('sitePortalOverviewPage'))`) helper?